randovania/data/json_data/prime3.txt

```diff
@@ -3659,10 +4511,13 @@ Asset id: 11846340785047609812
   > Dock to Scrapvault Lift
       Trivial
   > Dock to Processing Access
-      Trivial
+      After Scrapvault Wall
   > Pickup (Missile Expansion)
       Use Screw Attack (Space Jump)
 
+> Event - Open Gate; Heals? False
+  * Event Scrapvault Wall
+
```

Comment:
Connect this node to xray terminal morph door (as stated above) as trivial requirement.